posthog/api/signup.py

```diff
@@ -491,27 +511,42 @@ def social_create_user(
     *args,
     **kwargs,
 ):
+    invite_id = strategy.session_get("invite_id")
+    backend_processor = "social_create_user"
+    email = details["email"][0] if isinstance(details["email"], list | tuple) else details["email"]
```

Comment:
This `details` dict is something that comes in a request? I am thinking about security issues. Is it possible they send a different email str in the request, and then login with an email they don't have access to? I know it would be too much of a workaround (they would have to have access to another email in that SSO organization), but just curious.

Example:

I have `a@org.com`, and then in the request details i send `b@org.com`, fetching `b@org.com` invitation.

Reply:
This is the callback after the `saml/complete` so it's already authenticated the call from IdP configured on the organization domain.